packages/x-components/src/components/currency/base-currency.vue

```diff
@@ -6,11 +6,28 @@
   import Vue from 'vue';
   import { Component, Prop } from 'vue-property-decorator';
   import { currencyFormatter } from '../../utils/currency-formatter';
+  import { XInject } from '../decorators/injection.decorators';
 
   /**
-   * Renders the value received as a property, which always must be a JavaScript number, with the
-   * proper format provided as string property. The rendered tag is a span in order to render a
-   * default inline HTML element.
+   * Renders the value received as a property which always must be a JavaScript number, with the
+   * proper format provided as string property or by injection. The rendered tag is a span in order
+   * to render a default inline HTML element.
+   *
+   * Regarding the format or mask to be defined as string:
+   * - Use 'i' to define integer numbers.
+   * - Use 'd' to define decimal numbers. You can define the length of the decimal part. If the
+   * doesn't include decimals, it is filled with zeros until reach the length defined with 'd's.
```

Comment:
` * format doesn't include decimals, it is filled with zeros until reach the length defined with 'd's.`